src/resumable-task-store-sql.ts

```diff
@@ -111,11 +111,21 @@ export class ResumableTaskStoreSql implements ResumableTaskStore {
       throw new Error('Connection to database not open. Call `open` before using `read`.');
     }
 
-    return this.#db
+    const task = await this.#db
       .selectFrom('resumableTasks')
       .selectAll()
       .where('id', '=', taskId)
       .executeTakeFirst();
+
+    if (task !== undefined) {
+      // NOTE: special handling ONLY for PostgreSQL:
+      // Even though PostgreSQL stores `bigint` as a 64 bit number, the `pg` library we depend on returns it as a string, hence the conversion.
```

Comment:
Yikes! This must have been a pain to track down!

Reply:
😭